scripts/upgrade_kernel.sh

```diff
@@ -13,13 +13,15 @@ if [[ -z "$KERNEL_VERSION" ]]; then
   echo "kernel_version is unset. Setting to $KERNEL_VERSION based on Kubernetes version $KUBERNETES_VERSION."
 fi
 
-if [[ $KERNEL_VERSION == "4.14" ]]; then
-  sudo yum update -y kernel
+if [[ $KERNEL_VERSION == 4.14* ]]; then
+  sudo yum install -y kernel-${KERNEL_VERSION}
```

Comment:
Have you verified that this still works?

Reply:
I need to tack a `*` on the end, then I'll kick off a CI run for it